### PR TITLE
Feature/db template

### PR DESCRIPTION
**Pull request by CookieDecide**
Added a new folder for database models and implemented a basic template for peewee models.

### CODE REVIEW

File: src/model/db_template.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+import peewee as pw
+import os
+
+# check if db folder exists
+if not os.path.exists("../db"):
+    os.mkdir("../db")
+
+# link the database file
+TEMPLATE_DB = pw.SqliteDatabase("../db/template.db", check_same_thread=False)
```
Comment (Darkuuu) [inline src/model/db_template.py:13]: Does this make sense to execute this code always when this module is imported? Wouldn't it be better to implement a Singleton (or does pw take care of doing that)?

Additionally, I would make the string "../db/template.db" as a field on top of that class to make it easier to change.

Comment (CookieDecide) [reply src/model/db_template.py:13]: How would you go about implementing that as a singleton?

Comment (Darkuuu) [reply src/model/db_template.py]: Clarified in call. Will go with this approach for now, if problematic will be fixed later.

**Review by Darkuuu — APPROVED** [review]
lgtm

**Pull request merged**